scripts/flush-dev-db.sh

```diff
@@ -1,18 +1,51 @@
 #!/bin/bash
-
 set -euo pipefail
 
+function check_dep {
+  if ! command -v "${1}" &>/dev/null; then
+    echo "You need '${1}' - '${2}'"
+    exit 1
+  fi
+}
+
+# Check dependencies
+check_dep "psql" "brew install postgresql"
+check_dep "jq" "brew install jq"
+check_dep "aws" "brew install awscli"
+check_dep "kubectl" "asdf install kubectl 1.19.15"
+
+# check for the ENV variable
+set +u
+if [[ ! "${ENV}" =~ ^(dev|preprod)$ ]]; then
```

Comment:
Personally I'd prefer this and `port-forward-db` to both use a CL arg for env not an environment variable - but also they should be consistent with one another.